gammapy/background/reflected.py

```diff
@@ -309,12 +313,28 @@ def __str__(self):
         s += "\n{}".format(self.finder)
         return s
 
+    @staticmethod
+    def _get_bounding_size(region, center):
+        """Return the typical radius of the bounding size"""
+
+        geom = WcsGeom.create(skydir=center, width=10.*u.deg, binsz=Angle("0.01 deg"), coordsys="GAL", proj="TAN")
+
```

Comment:
After testing, this fails for regions and/or positions that located close to the poles because the `ixmax`and `ixmin`
values are not defined.
For safety a test should be made to determine the most relevant coordinate system to use.